.circleci/config.yml

```diff
@@ -0,0 +1,182 @@
+version: 2.1
+
+orbs:
+  win: circleci/windows@4.1
+
+jobs:
+  test_linux:
+    parameters:
+      ruby_version:
+        description: "version tag for the veracross/circleci-ruby-freetds container"
+        type: string
+
+    # Primary container image where all commands run
+    docker:
+      - image: cimg/ruby:<< parameters.ruby_version >>
+        environment:
+          TESTOPTS: "-v"
+          TINYTDS_UNIT_DATASERVER: sqlserver
+          TINYTDS_UNIT_HOST: sqlserver
+          SA_PASSWORD: super01S3cUr3
+          TOXIPROXY_HOST: toxiproxy
+
+      # Service container image
+      - image: mcr.microsoft.com/mssql/server:2019-CU16-ubuntu-20.04
+        name: sqlserver
+        environment:
+          ACCEPT_EULA: Y
+          SA_PASSWORD: super01S3cUr3
+
+      - image: shopify/toxiproxy:2.1.4
```

Comment:
When I looked back, this was the version used the last time those tests worked. I believe newer versions will break the tests more. 

Is it alright if we leave toxiproxy for later? I feel it makes it clear for the next person what version was last used for those tests. And I hate to pin a version of something I am fairly confident won't work at all without updated tests. 

Otherwise, I can pin it to the latest version, but the toxiproxy stuff will all got to get addressed at some point. Let me know.

Reply:
How about this: switch to the GH registry, but keep the reference to this version, and add a brief comment explaining that it's on purpose

Reply:
@aharpervc, does it matter what organization the final circleci pipeline is registered under? I ask since the shield committed will point to this.